mitodl/hugo-course-publisher
# ocw to hugo 1.0.7

#### Pre-Flight checklist

#### What are the relevant tickets?
Closes https://github.com/mitodl/hugo-course-publisher/issues/248

#### What's this PR do?
This PR updates this project to use `ocw-to-hugo` 1.0.7.  The `import:ocw` command has also been simplified and consolidated to use a single command and environment variables for the arguments used in `ocw-to-hugo`.  The environment variables used are documented in README.md.

#### How should this be manually tested?
Read the readme changes and make sure you can use environment variables to successfully import courses and run a site using the `import:ocw` package script.
